README.md

```diff
@@ -1,151 +1,72 @@
 
-kops is the easiest way to get a production Kubernetes up and running.  We like to think
-of it as "kubectl for clusters".
+The easiest way to get a production Kubernetes cluster up and running.  
 
-Some of the more interesting features:
+# What is kops?
 
-* Written in go, so hopefully easier to maintain and extend, as complexity inevitably increases
-* Uses a state-sync model, so we get things like a dry-run mode and idempotency automatically
-* Based on a simple meta-model defined in a directory tree
-* Can produce configurations in other formats (currently Terraform & Cloud-Init), so that we can have working
-  configurations for other tools also.
-
-## Recent changes
-
-Change history is available in [HISTORY.md](HISTORY.md)
-
-## Installation
-
-We recommend using a release from the Release tab above.
-
-## Installation from source
-
-Build the code (make sure you have set GOPATH):
-```
-go get -d k8s.io/kops
-cd ${GOPATH}/src/k8s.io/kops/
-git checkout release
-make
-```
-
-* The `release` branch is where releases are taken from.  This is the stable code branch.
-* The `master` branch  _should_ also be functional, but is where active development happens, so may be less stable.
-
-(Note that the code uses the relatively new Go vendoring, so building requires Go 1.6 or later,
-or you must export GO15VENDOREXPERIMENT=1 when building with Go 1.5.  The makefile sets
-GO15VENDOREXPERIMENT for you.  Go code generation does not honor the env var in 1.5, so for development
-you should use Go 1.6 or later)
-
-## Bringing up a cluster on AWS
-
-* Ensure you have kubectl installed and on your path.  (We need it to set kubecfg configuration.)
-
-* Set up a DNS hosted zone in Route 53, e.g. `mydomain.com`, and set up the DNS nameservers as normal
-  so that domains will resolve.  You can reuse an existing domain name (e.g. `mydomain.com`), or you can create
-  a "child" hosted zone (e.g. `myclusters.mydomain.com`) if you want to isolate them.  Note that with AWS Route53,
-  you can have subdomains in a single hosted zone, so you can have `cluster1.testclusters.mydomain.com` under
-  `mydomain.com`.
+We like to think of it as `kubectl` for clusters. 
 
-* Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster
-  can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be
-  `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+The tool enables the user to deploy functional Kubernetes clusters in the cloud ([AWS](/docs/aws.md) or GCE). 
 
-* Set `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+We encourage everyone to read more about [how it works](/docs/how_it_works.md) or check out some of the [other interesting modes](/docs/commands.md#other-interesting-modes).
```

Comment:
Can we a link to aws.md more prominent.  It is why people are here.

Maybe: "Quickstart: Launching a kubernetes cluster on AWS"